libs/server-sdk/src/data_systems/background_sync/sources/streaming/streaming_data_source.hpp

```diff
@@ -6,50 +6,44 @@
 #include "../../../../data_interfaces/destination/idestination.hpp"
 #include "../../../../data_interfaces/source/idata_synchronizer.hpp"
 
-#include <launchdarkly/server_side/config/built/all_built.hpp>
-
 #include <launchdarkly/logging/logger.hpp>
+#include <launchdarkly/server_side/config/built/all_built.hpp>
 #include <launchdarkly/sse/client.hpp>
 
 #include <boost/asio/any_io_executor.hpp>
 
-#include <chrono>
-
-using namespace std::chrono_literals;
-
 namespace launchdarkly::server_side::data_systems {
 
 class StreamingDataSource final
     : public data_interfaces::IDataSynchronizer,
       public std::enable_shared_from_this<StreamingDataSource> {
    public:
     StreamingDataSource(
-        config::built::ServiceEndpoints const& endpoints,
-        config::built::BackgroundSyncConfig::StreamingConfig const&
-            data_source_config,
-        config::built::HttpProperties http_properties,
-        boost::asio::any_io_executor ioc,
-        data_interfaces::IDestination& handler,
+        boost::asio::any_io_executor io,
+        Logger const& logger,
         data_components::DataSourceStatusManager& status_manager,
-        Logger const& logger);
+        config::built::ServiceEndpoints const& endpoints,
+        config::built::BackgroundSyncConfig::StreamingConfig const& streaming,
+        config::built::HttpProperties const& http_properties);
 
-    void Init(std::optional<data_model::SDKDataSet> initial_data) override;
-    void StartAsync() override;
+    void StartAsync(data_interfaces::IDestination* dest,
+                    data_model::SDKDataSet const* bootstrap_data) override;
     void ShutdownAsync(std::function<void()> completion) override;
 
     [[nodiscard]] std::string const& Identity() const override;
 
    private:
```

Comment:
To match the constructor args, the members are re-arranged (so the order of initialization is visually correct.)